disable SSE intrinsics to avoid error when compiling with Intel C++ compiler:
"error: expression must have integral or enum type"
in genIBS.cpp and genKING.cpp due to use of complement (~) on values of type _m128i
author: Kenneth Hoste; patch suggested by Xiuwen Zheng (SNPRelate maintainer)
--- SNPRelate.orig/src/Makevars	2016-07-29 06:39:00.000000000 +0200
+++ SNPRelate/src/Makevars	2016-08-26 09:31:44.440787000 +0200
@@ -3,7 +3,7 @@
 ###                                                            ###
 
 # additional preprocessor options
-PKG_CPPFLAGS = -DUSING_R -I.
+PKG_CPPFLAGS = -DUSING_R -I. -DCOREARRAY_NO_SIMD
 
 # to set flags for the linker
 PKG_LIBS = $(FLIBS) $(BLAS_LIBS) $(LAPACK_LIBS) -lpthread
